packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -29,7 +30,7 @@
 )
 from .projects_nodes_ui import Position
 from .projects_state import RunningState
-from .services import PROPERTY_KEY_RE, ServiceKey, ServiceVersion
+from .services import ServiceKey, ServiceVersion
 
 # NOTE: WARNING the order here matters
```

Comment:
when I see the positioning of this warning, I think this lost its initial location. but I guess this has to do with the union types below..
There are also a FIXME and a TODO below (line 41  and 52)

Reply:
This is for the union, yes